deepali/losses/functional.py

```diff
@@ -636,6 +639,83 @@ def ssd_loss(
     return loss
 
 
+def mi_loss(
+        input: Tensor,
+        target: Tensor,
+        mask: Optional[Tensor] = None,
+        vmin: Optional[float] = None,
+        vmax: Optional[float] = None,
+        num_bins: int = 64,
+        sample_ratio: Optional[float] = None,
+        nmi: bool = True
```

Comment:
Given that the function name includes a reference to "mutual information" (MI) already, I would suggest renaming this keyword argument to "normalized" or "normalize".

Reply:
Agreed. Changing to `normalized`